app/lib/zammad_api_client.rb

```diff
@@ -0,0 +1,50 @@
+class ZammadApiClient
+  class Error < StandardError; end
+
+  def self.connection
+    Faraday.new(
```

Comment:
```suggestion
  def self.connection
    @connection ||= Faraday.new(
```

Je pense d’ici on peut mémoïzer

Reply:
merci ! j’ai appliqué ce changement même si j’ai un léger doute / brouillard sur ce que ça veut dire en termes de parallélisme de partager la connexion